### DISCUSSION TITLE
GPS Issues

### DISCUSSION THREAD

**Discussion by GutHib** [organicmaps/organicmaps#5061] [Q&A] [answered]
Hi all,

I am having continuous problems getting OM to show my GPS position within reasonable time. My phone is a Samsung S5 phone running LineageOS, and I would be more than happy to blame my particular setup, but when i open Google maps in my phones browser (i don't have the actual GM installed), i get the position fairly quickly. I am assuming that Google maps is using the same hardware/tools on my phone to come up with the position, so i don't understand why there's such a big difference. OM will eventually find me, too, but when you're traveling, it's just not feasible to stand around in some shady neighborhood for ten minutes waiting for directions. Has anyone had similar issues (or, even better, found a way to fix them)?

Thanks!

**Comment by hb0nd** [comment]
https://github.com/organicmaps/organicmaps/pull/5022 may fix it in the next release. Do you have MicroG (UnifiedNLP)? that may also help 😄

**Reply by GutHib** [reply]
Thanks for pointing that out - are there nightly snapshots or do i need to wait until the next official release?

I do have microg installed and was wondering if it's best to choose one location module that works or if you should simply install all your can lay your hands on?

**Reply by hb0nd** [reply, marked as answer]
> I do have microg installed and was wondering if it's best to choose one location module that works or if you should simply install all your can lay your hands on?

MicroG requests a location from all modules at the same time, then (AFAIK) uses whatever comes first, so having more will result in more requests but possibly faster fix :) personally I use 'Apple Wi-Fi' & 'Mozilla Location Service'


in terms of nightly snapshots, there are beta builds here: https://t.me/s/OrganicMapsApp and on Google Firebase: https://appdistribution.firebase.google.com/pub/i/9ec3bca5e2b47373, however there hasn't been a build that includes the location fix commit yet :)

**Reply by GutHib** [reply]
Thanks - much appreciated.
